<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>808</width>
    <height>860</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblmaintext">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>451</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Método de condensación</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="txb11">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txb12">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>110</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txb13">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>110</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txb14">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>110</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txb16">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>110</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txb17">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>110</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txb15">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>110</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txb25">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>200</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txb27">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>200</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txb23">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>200</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txb24">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>200</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txb26">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>200</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txb22">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>200</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txb21">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>200</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txb35">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>290</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txb37">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>290</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txb46">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>380</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txb43">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>380</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txb33">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>290</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txb34">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>290</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txb47">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>380</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txb42">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>380</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txb36">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>290</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txb44">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>380</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txb45">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>380</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txb32">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>290</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txb41">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>380</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txb31">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>290</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txb62">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>560</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txb73">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>650</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txb66">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>560</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txb75">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>650</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txb57">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>470</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txb61">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>560</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txb77">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>650</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txb55">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>470</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txb52">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>470</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txb51">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>470</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txb53">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>470</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txb64">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>560</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txb74">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>650</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txb63">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>560</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txb56">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>470</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txb72">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>650</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txb76">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>650</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txb54">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>470</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txb67">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>560</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txb65">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>560</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="txb71">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>650</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btncalculate">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>750</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Calcular</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnexit">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>750</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblresult">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>760</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Resultado</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>808</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
